src/tools/compiletest/src/header.rs

```diff
@@ -948,6 +951,13 @@ impl Config {
                         raw,
                         testfile.display()
                     );
+                } else if FORBIDDEN_REVISION_NAMES.contains(&revision.as_str()) {
+                    panic!(
+                        "invalid revision: `{}` in line `{}`: {}",
```

Comment:
Suggestion: this is very uninformative, we should say something like (line break as suitable, probably)

```
revision name `{revision}` is not permitted in a test suite that uses `FileCheck` annotations as it is confusing when used as custom `FileCheck` prefix: `{revision}` in line `{}`:  {}
```